pkg/logbridge/logbridge.go

```diff
@@ -0,0 +1,116 @@
+package logbridge
+
+import (
+	"bufio"
+	"io"
+	"time"
+
+	"github.com/square/p2/pkg/logging"
+)
+
+// Copy implements a buffered copy operation between dest and src.
+// It returns the number of dropped messages as a result of insufficient
+// capacity
+// The caller is responsible for closing & flushing the reader and writer.
```

Comment:
> The caller is responsible for closing & flushing the reader and writer.

The caller is busy executing this method and won't be able to do anything to the reader or writer.  I think you're trying to say that something else should take care of making sure the reader and writer have enough data.  Don't worry about it.